File: Service_eHospital_DongNai_A_NSTL/Implement/NS_DM_TONGIAOService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_DM_TONGIAOService : BaseService<NS_DM_TONGIAO, INS_DM_TONGIAORepository>
    , INS_DM_TONGIAOService
    {
    private readonly INS_DM_TONGIAORepository _NS_DM_TONGIAORepository;
    public NS_DM_TONGIAOService(INS_DM_TONGIAORepository NS_DM_TONGIAORepository) : base(NS_DM_TONGIAORepository)
    {
    _NS_DM_TONGIAORepository = NS_DM_TONGIAORepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_DM_TONGIAO.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_DM_TONGIAO } from './NS_DM_TONGIAO.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_DM_TONGIAOService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'MaTonGiao', 'TenTonGiao'];

    List: NS_DM_TONGIAO[] | undefined;
    ListFilter: NS_DM_TONGIAO[] | undefined;
    FormData!: NS_DM_TONGIAO;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_DM_TONGIAO";
    }
}

